README.md

```diff
@@ -27,14 +27,14 @@ These videos on TikTok, YouTube and Instagram get MILLIONS of views across all p
 
 - Python 3.6+
 - Playwright (this should install automatically during installation)
-
+- pip
 ## Installation 👩‍💻
 
 1. Clone this repository
 2. Rename `.env.template` to `.env` and replace all values with the appropriate fields. To get Reddit keys (**required**), visit [the Reddit Apps page.](https://www.reddit.com/prefs/apps) TL;DR set up an app that is a "script". Copy your keys into the `.env` file, along with whether your account uses two-factor authentication.
-3. Run `pip3 install -r requirements.txt`
-4. Run `playwright install` and `playwright install-deps`.
-5. Run `python3 main.py`
+3. Open CMD in the directory and run `pip3 install -r requirements.txt`
```

Comment:
This should be independent of platform. CMD is only for Windows, changing it to something like 'the terminal' or 'the command line' would be a better term